admin/tests-integration/check-form.service.spec.js

```diff
@@ -1,84 +0,0 @@
-'use strict'
-
-/**
- * @file Integration Tests for Check Form Service
- */
-
-/* global describe test expect beforeAll jest */
-
-// These tests take a long time, so they are usually skipped unless we change the implementation
-jest.setTimeout(120000)
-
-const fs = require('fs')
-const moment = require('moment')
-
-const checkFormService = require('../services/check-form.service')
-
-describe.skip('check-form.service', () => {
```

Comment:
I don't advise deleting this file - it _ought_ to remain and be re-ran whenever allocateCheckForm() changes.   It does appear to be in a bit of a limbo situation.   We might want to just comment out the `writeToCsV` part and run a sense check on the accumulated form totals to make the test an actual integration test.

It's testing critical code to the check.

Reply:
have put it back and will override inline